src/Tests/Microsoft.NET.Sdk.Razor.Tests/AspNetSdkBaselineTest.cs

```diff
@@ -15,6 +15,36 @@
 using Xunit;
 using Xunit.Abstractions;
 
+
+/***
+
+-----------------------------------------------------------------
+  Blazor WebAssembly Static Web Asset (SWA) Baseline Generation
+-----------------------------------------------------------------
+
+The Blazor WASM SWA baselines are used to determine if the expected assets in 
+the manifest are present, and if new assets have been added. Occasionally, it's 
+required to update the SWA baselines to account for expected changes in the 
+packaged files.
+
+**Please note:** These steps must be performed in a Windows development environment
+otherwise the line-ending and ordering result in very large diff-counts.
+
+
+Updating Baselines:
+
+1. Restore repo: `.\restore.cmd`
+2. Build dotnet: `.\build.cmd`
+3. Activate local dotnet environment: `.\eng\dogfood.cmd`
+4. Enter the following into the CLI from the activated dogfood environment to open VS: `.\src\BlazorWasmSdk\BlazorWasmSdk.slnf`
+5. Add `#define GENERATE_SWA_BASELINES` on the first line of this file (above the license comment)
+6. Run all tests from `Microsoft.NET.Sdk.BlazorWebAssembly.Tests.csproj`
+7. Delete `#define GENERATE_SWA_BASELINES` added in step 5 from the top of this file.
+8. Commit the changes to the project baselines.
+
+ ***/
+
+
```

Comment:
This is obsolete, I automated this over the weekend and there is a script you can run in src/RazorSdk/update-template-baselines.ps1 or something like that

Reply:
https://github.com/dotnet/sdk/blob/main/src/RazorSdk/update-test-baselines.ps1